services/orchest-api/start.sh

```diff
@@ -11,5 +11,5 @@ python migration_manager.py db upgrade
 if [ "$FLASK_ENV" = "development" ]; then
     python main.py
 else
-    exec gunicorn -k eventlet -c "$GUNICORN_CONF" "$APP_MODULE"
+    exec gunicorn -w 1 --threads 100 -c "$GUNICORN_CONF" "$APP_MODULE"
```

Comment:
From the gunicorn docs:
> This setting only affects the Gthread worker type.
> (...)
> If you try to use the sync worker type and set the threads setting to more than 1, the gthread worker type will be used instead.

Given the docs, I think it would make more sense to be explicit about the worker type that is used, thus changing it to
```suggestion
    exec gunicorn -k gthread --threads 100 -c "$GUNICORN_CONF" "$APP_MODULE"
```

Reply:
Agree